src/backend/utils/misc/pipelinedb.conf.sample

```diff
@@ -627,6 +627,16 @@
 # each database
 #continuous_query_num_workers = 2
 
+# allow direct changes to be made to materialization tables?
+#continuous_query_materialization_table_updatable = off
```

Comment:
Any reason to do so? Only power users ever really look at configuration options and it's prolly ok to expose this to them?